docs/source/02_Terminology.md

```diff
@@ -145,24 +145,25 @@ The types of characters referred to in this specification are:
 | Name | Description |
 | ---- | ----------- |
 | `ascii` | utf-8 codes 0 to 127 (single byte) |
-| `non-ascii` | utf-8 codes greater than 128 (multi-byte) |
+| `nonascii` | utf-8 codes greater than 128 (multi-byte) |
 | `printable` | ASCII 32 <= code < 127 |
 |`lowercase` | ASCII characters a-z |  
 | `uppercase` | ASCII characters A-Z |
 | `letters` | `lowercase` and/or `uppercase` |
-| `text` | `printable` and/or `non-ascii` |
+| `text` | `printable` and/or `nonascii` |
 | `digits` | 0-9 |
 | `tab` | ASCII code 09 |
 | `newline` | ASCII code 10 (linefeed) |
 | `blank` | ASCII code 32 |
 | `number-sign` | ASCII code 35 |
+| `dollar` | ASCII code 36 |
```

Comment:
Is this used for anything?

Reply:
Yes, it is going to be used as an allowedCharacter in various value classes and unit classes in 8.3.0.